frontend/ui/widget/buttondialog.lua

```diff
@@ -146,20 +194,36 @@ function ButtonDialog:init()
             VerticalSpan:new{ width=Size.padding.buttontable },
         }
     end
+    local separator
+    if self.title then
+        separator = LineWidget:new{
+            background = Blitbuffer.COLOR_GRAY,
+            dimen = Geom:new{
+                w = self.buttontable:getSize().w + (scrollbar_width or 0),
+                h = Size.line.medium,
+            },
+        }
+    else
+        separator = VerticalSpan:new{}
+    end
     self.movable = MovableContainer:new{
-            alpha = self.alpha,
-            anchor = self.anchor,
-            FrameContainer:new{
+        alpha = self.alpha,
+        anchor = self.anchor,
+        FrameContainer:new{
+            VerticalGroup:new{
```

Comment:
When instantiating a widget, we usually (I think?) put first all the properties (radius=..., background=...) and only then the subwidgets, so they usually appear last.